lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -77,34 +85,20 @@ def self.keywords
         rule %r/\s+/, Text::Whitespace
         rule %r/\)/, Punctuation, :pop!
         rule %r/\(/, Punctuation
-        rule %r/[a-z0-9\/_.]*/i, Str
+        rule %r/[a-z0-9\/_.-]*/i, Str
```

Comment:
Should the character range be expanded further to include other characters like `%`, `[`, `]` and `+`?

Reply:
I suppose it could contain anything a filename can contain. In practice this range seems to be fine, but it's just a path to a filename which could be all kinds of things so it wouldn't hurt to expand this.

Reply:
I went ultra-liberal and match everything except whitespace.

I also made it so that using a `(` in a filename requires you to close it with a `)` in order to exit the `:includepath` state. Let me know if you think this is the wrong track to take.

Reply:
Stupid me. Having `(` add another level to the stack means that you never fully exit this state. I think we just assume that nobody is going to put `(` or `)` in include paths.

Reply:
Yes, no way to put `(` or `)` in an include unless they are in the file name and escaped, but I hope nobody is doing something so ugly.